src/components/BookmarkPage.js

```diff
@@ -3,16 +3,47 @@ import RestaurantBox from "./RestaurantBox";
 import { ReactComponent as Find } from "../icons/find.svg";
 import { Link } from "react-router-dom";
 
-export default function BookmarkPage() {
+export default function BookmarkPage({ restaurantData }) {
+  const bookmarkedRestaurants = restaurantData.filter(
+    (savedRestaurants) => savedRestaurants.isBookmarked === true
+  );
+
+  function renderRestaurants() {
+    return bookmarkedRestaurants.map((restaurant) => {
+      const {
+        id,
+        name,
+        kitchen,
+        location,
+        atmosphere,
+        isBookmarked,
+      } = restaurant;
```

Comment:
Very good, destructuring for the win. 

Just a curiosity: did you know you can directly destructure in the function definition? Like this:

```js
    return bookmarkedRestaurants.map(({ id, name, kitchen, location, atmosphere, isBookmarke }) => {
         // Then you can use it  right away
         console.log(name);
    }
```

This is exactly what we do in react components to destructure the `props`

Reply:
Yes true, looks better like that